### ISSUE TITLE
[Suggestion] When attributes have different names

### ISSUE THREAD

**Issue by nelfer** [jitbit/AspNetSaml#2]
Hi!
Thanks for this project. It really simplified my life.
I've tested it and it works great.

But I have a suggestion, and I think you kinda started implementing it, which is, when the attribute name in the assertion is not what you expect or different identity providers have different names.

The way I'm solving it, I have a small class to hold "attribute name" and "attribute value number" (in case is a multi-value attribute) and then just changed this line:

```
AttributeConfig a = sm.GetUserGroupAttribute();
XmlNode node = _xmlDoc.SelectSingleNode("/samlp:Response/saml:Assertion/saml:AttributeStatement/saml:Attribute[@Name='"+a.Name+"']/saml:AttributeValue["+a.Value+"]", _xmlNameSpaceManager);
```
I have a helper class (sm above) that reads from configuration and retrieves the configured name. This allows to configure for Azure (for example) which uses a different name. And testing with another identity provider, I noticed a different name as well.

You could put those helper methods "GetUserGroupAttribute", "GetUserFirstName", etc. as part of the class or as a helper class returning some fixed values and then it will be up to the implementer to modify it in order to retrieve it from configuration.

Thanks

**Comment by alex-jitbit** [comment]
I like this. Gonna think about it

**Comment by salihkiraz** [comment]
hi @nelfer https://github.com/jitbit/AspNetSaml/pull/31

**Comment by alex-jitbit** [comment]
fixed in #31
